### ISSUE TITLE
[ext] Namibia dashboard summary table - separate % Refused and % Locked columns by first visit vs. mop-up

### ISSUE THREAD

**Issue by TNgidi** [locational/douma-app#461]
@onlyjsmith commented on [Thu Jul 19 2018](https://github.com/locational/_tickets/issues/281)

_External feedback source:_ https://github.com/disarm-platform/user-requests-and-feedback/issues/87
Currently we have the following 2 columns on the far right of the dashboard:

% Refused (# structures unsprayed where Reason == Refused) and 
% Locked (# structures sprayed where Reason == Locked)

We want to split this into:

% Refused First Visit (# structures unsprayed (first visit) where Reason == Refused) 
% Refused Mop-up (# structures unsprayed (mop-up) where Reason == Refused) 
% Locked First Visit (# structures sprayed (first visit) where Reason == Locked)
% Locked Mop-up (# structures sprayed (mop-up) where Reason == Locked)

This is dependent on #86 where there will be separate questions for number unsprayed first visit vs. number unsprayed mop-up

**Comment by Nicolaidavies** [comment]
@TNgidi I updated the aggregations and the new config is live on stage as version 1.0.15.

**Comment by TNgidi** [comment]
Thanks for the information @Nicolaidavies, I will test now

**Comment by TNgidi** [comment]
I don't know if the coverage calculations are correct. is the screen show below I think we should show atleast 0.0.... something because there structures sprayed
![calculations](https://user-images.githubusercontent.com/36029281/43248750-1f0fb0c4-90b9-11e8-91ae-b15ac1695ec7.png)

**Comment by TNgidi** [comment]
![calculations](https://user-images.githubusercontent.com/36029281/43249011-d7ee4312-90b9-11e8-9a81-15020be35010.png)

**Comment by Nicolaidavies** [comment]
Strange, I haven't touched anything with the coverage. The refused and locked will show up as 0 until we create new `fake_forms`.

**Comment by Nicolaidavies** [comment]
@TNgidi Please take a look at the instance `tarajuke-287` it has coverage showing up correctly.

**Comment by TNgidi** [comment]
@Nicolaidavies On this instance “testingcoverage” I have added some records with structures that match the columns on the table (locked first visit, locked mop up, refused first visit, refused mop up) but the table still shows 0 %

**Comment by TNgidi** [comment]
@Nicolaidavies is OK to add records for more testing to your instance ID tarajuke-287 ?

**Comment by Nicolaidavies** [comment]
This change was deployed to staging and production. 

Confirmed working by Sameen. Moving to 'Review/QA'

**Comment by TNgidi** [comment]
Done
